pkg/runtime/wfengine/backends/actors/activity_actor.go

```diff
@@ -364,6 +393,13 @@ func (a *activityActor) createReliableReminder(ctx context.Context, data any) er
 	if err != nil {
 		return fmt.Errorf("failed to encode data as JSON: %w", err)
 	}
+
+	if a.actorRuntime.UsingSchedulerReminders() == true {
```

Comment:
This is an abstraction violation. The WF actors should work regardless of the scheduler service. Does it mean that actors that implemented in the app will have the problem with reminders not being deleted too if they also don't have this abstraction violation?